### DISCUSSION TITLE
Fix Visuliazer that built vis_backends will not be used when save_dir is None 修复 Visualizer 的 save_dir 参数为 None 时，传入构建好的 vis_backends 不起作用的问题

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#593] [MMEngine]
#  Fix Visuliazer that built vis_backends will not be used when save_dir is None 
 修复 Visualizer 的 save_dir 参数为 None 时，传入构建好的 vis_backends 不起作用的问题 

[https://github.com/open-mmlab/mmengine/issues/1267](https://github.com/open-mmlab/mmengine/issues/1267) 

[https://github.com/open-mmlab/mmengine/issues/1267](https://github.com/open-mmlab/mmengine/issues/1267) 

## Task Information

**Repository**：MMEngine

**Difficulty**：Easy

**Points**：30

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：None

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by Crescent-Saturn** [comment]
I think this task has already been completed by this PR: https://github.com/open-mmlab/mmengine/pull/1275
